src/item-view.js

```diff
@@ -12,35 +12,34 @@ Marionette.ItemView = Marionette.View.extend({
     Marionette.View.apply(this, arguments);
   },
 
-  // Serialize the model or collection for the view. If a model is
-  // found, the view's `serializeModel` is called. If a collection is found,
-  // each model in the collection is serialized by calling
-  // the view's `serializeCollection` and put into an `items` array in
-  // the resulting data. If both are found, defaults to the model.
-  // You can override the `serializeData` method in your own view definition,
-  // to provide custom serialization for your view's data.
-  serializeData: function(){
-    if (!this.model && !this.collection) {
-      return {};
-    }
+  // Serialize the view's model *or* collection, if
+  // it exists, for the template
+  serializeData: function() {
+    var data = {};
 
-    var args = [this.model || this.collection];
-    if (arguments.length) {
-      args.push.apply(args, arguments);
+    // If we have a model, we serialize that
+    if (this.model) {
+      data = this.serializeModel();
     }
 
-    if (this.model) {
-      return this.serializeModel.apply(this, args);
-    } else {
-      return {
-        items: this.serializeCollection.apply(this, args)
+    // Otherwise, we serialize the collection,
+    // making it available under the `items` property
+    else if (this.collection) {
+      data = {
+        items: this.serializeCollection()
       };
     }
+
+    return data;
   },
 
-  // Serialize a collection by serializing each of its models.
-  serializeCollection: function(collection){
-    return collection.toJSON.apply(collection, _.rest(arguments));
+  // Serialize a collection by cloning each of
+  // its model's attributes
+  serializeCollection: function() {
+    if (!this.collection) { return {}; }
+    return this.collection.map(function(model) {
```

Comment:
Damn, I misspoke. For some reason, I thought it `Model` just used `attributes`, instead of [setting](https://github.com/jashkenas/backbone/pull/873/files#diff-0d56d0d310de7ff18b3cef9c2f8f75dcR165) it. :disappointed: